### ISSUE TITLE
Calendar Add/Edit Event

### ISSUE THREAD

**Issue by trf000** [AppStateESS/phpwebsite#203]
Use the same datepicker as Blog. 

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/197608664378684/197609695648823)

**Comment by Stardog** [comment]
It is using datetimepicker (although was broken before #311be2b94194d1f0b0e3280a2a7c97eafdbd4685). Did you want the time choice to work like Blog instead of using drop downs?

**Comment by trf000** [comment]
Yes, i think it looks better and keeps a sense of uniformity across modules.

On Mon, Feb 16, 2015 at 12:05 PM, Matthew McNaney <notifications@github.com>
wrote:

> It is using datetimepicker (although was broken before
> #311be2b94194d1f0b0e3280a2a7c97eafdbd4685). Did you want the time choice to
> work like Blog instead of using drop downs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AppStateESS/phpwebsite/issues/203#issuecomment-74549462>
> .
>



-- 
Pretend inferiority and encourage his arrogance.

**Comment by Stardog** [comment]
I started on this. Not as easy a fix as originally thought, especially when bringing in All Day. I will return to this but for now I branched it.
